.circleci/config.yml

```diff
@@ -243,7 +243,7 @@ jobs:
           path: c:\tmp\test-reports
   test-ui:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/node:12-browsers
+      - image: docker.mirror.hashicorp.services/circleci/node:14-browsers
```

Comment:
`question`: should we review the changelog in Node to are deprecations or breaking changes that we should consider?

What's the surface area of this change?

Reply:
I think we're good here. I took a look at it quickly a while back and didn't see anything relevant. We also use node as a built tool, not the runtime, so any breaking change would likely cause the UI to not even build.